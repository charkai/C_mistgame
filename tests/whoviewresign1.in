place C9
view
who
place S19
view
who
resign
