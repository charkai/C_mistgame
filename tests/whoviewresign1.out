B16,xx.....xx.....xx.....xx.....xx.....xx.....xx.....
W
E16,.................................................
B
White wins!
C9S19
Thank you for playing!
